buildscripts/include/download-sdk.sh

```diff
@@ -68,7 +68,7 @@ if [ -d "android-ndk-${v_ndk}" ]; then
 	:
 elif [ -d "android-sdk-$os/ndk/${v_ndk_n}" ]; then
 	ln -s "android-sdk-$os/ndk/${v_ndk_n}" "android-ndk-${v_ndk}"
-elif [ -z "${os_ndk}" ]; then
+elif [ -d "android-sdk-${os}" ] && [ ! -d "android-sdk-$os/ndk/${v_ndk_n}" ]; then
```

Comment:
> but this would then not work for the darwin case

why not?
`os_ndk` is *not* set so it enters this branch and downloads the NDK via sdkmanager
it then creates a symlink so the ndk is found at the expected location
this already works and there is no need to change it.